user_tools/src/spark_rapids_tools/utils/util.py

```diff
@@ -323,3 +323,16 @@ def bytes_to_human_readable(cls, num_bytes: int) -> str:
             num_bytes /= 1024.0
             i += 1
         return f'{num_bytes:.2f} {size_units[i]}'
+
+    @classmethod
+    def convert_to_camel_case(cls, input_str: str) -> str:
```

Comment:
I remember we had some utilities to convert to camel_case 
IIRC, there was `convert_dict_to_camel_case` to convert entire dictionary, and other utilities like `to_camel_case`
If there is no difference in functionality, then let's keep only one version of them.

Reply:
Thanks @amahussein. Reused existing `convert_dict_to_camel_case()`